packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-spread/expression/exec.js

```diff
@@ -0,0 +1,8 @@
+var log = [];
+
+var a = {
+  ...{ get foo() { log.push(1); } },
+  get bar() { log.push(2); }
+};
+
+expect(log).toEqual([1]);
```

Comment:
Because [per the spec](https://www.ecma-international.org/ecma-262/9.0/index.html#sec-object-initializer-runtime-semantics-propertydefinitionevaluation) it copies property values by calling getters.